meet-data/apf/1503/entries.csv

```diff
@@ -0,0 +1,234 @@
+Name,Age,Sex,Division,Equipment,Event,BodyweightKg,WeightClassKg,Squat1Kg,Squat2Kg,Squat3Kg,BestSquatKg,Bench1Kg,Bench2Kg,Bench3Kg,BestBenchKg,Deadlift1Kg,Deadlift2Kg,Deadlift3Kg,BestDeadliftKg,TotalKg,Place,Team
+Richard Acuna,29,M,MORAAPF,Raw,SBD,106.6,110,192.5,217.5,-227.5,217.5,137.5,-142.5,-142.5,137.5,250,277.5,-282.5,277.5,632.5,2,
```

Comment:
This meet failed to distinguish between "Raw" and "Raw Assisted," so all the Raw lifters with an "S" event have to be pessimistically marked Wraps.